code/html-css/flexbox/style.css

```diff
@@ -0,0 +1,66 @@
+.challenge-1 {display: flex;flex-direction:row;
+    /* write code here */
+}
+
+.challenge-2 { display: flex;flex-direction: column-reverse; align-items: flex-end;
+    /* write code here */
+}
+
+.challenge-3 {display: flex;justify-content: right;
+    /* write code here */
+}
+
+.challenge-4 {display: flex;flex-direction: row;justify-content: space-evenly;
+    /* write code here */
+}
+
+.challenge-5 {display: flex;flex-direction:row ;justify-content: space-between;gap: 20px;
+    /* write code here */
+}
+
+/* 
+Tip: This is called the "child selector". 
+It selects all .bubble elements that are direct
+children of .challenge-6 elements. 
+*/
+.challenge-6 .bubble {display: flex;justify-content: center;align-items: center;
+    /* write code here */
+}
+
+.challenge-7 {
+    display: flex; flex-direction: row;
+}
+
+.challenge-7 .red {flex-grow: 1;width:100%;height: auto;border-radius: 0;
```

Comment:
Again, it's a very strange choice to write your CSS properties all on one line. And, why did you change the border radius? It doesn't hurt anything, but it doesn't help either, haha.